apps/hvn/package.json

```diff
@@ -21,19 +22,28 @@
     "@payloadcms/next": "^3.24.0",
     "@payloadcms/payload-cloud": "^3.24.0",
     "@payloadcms/richtext-lexical": "^3.24.0",
+    "@radix-ui/react-dropdown-menu": "^2.1.6",
+    "@radix-ui/react-label": "^2.1.2",
+    "@radix-ui/react-radio-group": "^1.2.3",
+    "@radix-ui/react-select": "^2.1.6",
+    "@radix-ui/react-slot": "^1.1.2",
     "@tanstack/react-query": "^5.66.9",
     "axios": "^1.7.9",
+    "class-variance-authority": "^0.7.0",
     "cross-env": "^7.0.3",
     "framer-motion": "^12.4.7",
     "graphql": "^16.10.0",
+    "lucide-react": "^0.378.0",
     "next": "^15.1.7",
     "next-intl": "^3.26.5",
+    "next-themes": "^0.4.4",
     "payload": "^3.24.0",
     "react": "^19.0.0",
     "react-dom": "^19.0.0",
     "react-hook-form": "^7.54.2",
     "react-icons": "^5.5.0",
     "sharp": "0.33.5",
+    "sonner": "^2.0.1",
     "stripe": "^17.6.0",
```

Comment:
https://payloadcms.com/docs/plugins/stripe

@myniqx I thought that this plugin helps with integrating.